components/supervisor/pkg/process/process_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package process
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/prometheus/procfs"
+	"github.com/stretchr/testify/require"
+)
+
+func TestVisitProcessTree(t *testing.T) {
```

Comment:
nit: should be a table driven test

Reply:
I'd argue there is already a second case: no child processes.
Having the table already encourages to add more cases.

Also, it's more in line with the [style guide](https://github.com/gitpod-io/go-style-guide/blob/master/style.md#test-tables) :)